gpytorch/priors/torch_priors.py

```diff
@@ -60,6 +60,21 @@ def expand(self, batch_shape):
         return UniformPrior(self.low.expand(batch_shape), self.high.expand(batch_shape))
 
 
+class HalfCauchyPrior(Prior, HalfCauchy):
+    """
+    Half-Cauchy prior.
+    """
+
+    def __init__(self, scale, validate_args=None, transform=None):
+        TModule.__init__(self)
+        HalfCauchy.__init__(self, scale=scale, validate_args=validate_args)
+        self._transform = transform
+
+    def expand(self, batch_shape):
+        batch_shape = torch.Size(batch_shape)
+        return HalfCauchy(self.loc.expand(batch_shape), self.scale.expand(batch_shape))
```

Comment:
No need for this, `Tensor.expand` works tuples, Sizes, variable # of args...
```suggestion
        return HalfCauchy(self.loc.expand(batch_shape), self.scale.expand(batch_shape))
```